amundsen_application/static/js/ducks/issue/api/v0.ts

```diff
@@ -0,0 +1,19 @@
+import axios, { AxiosResponse } from 'axios';
+
+export const API_PATH = '/api/issue';
+
+export function getIssues(tableKey: string) {
+  return axios.get(`${API_PATH}/issues?key=${tableKey}`)
+  .then((response: AxiosResponse) => {
```

Comment:
You can specify the type of the response payload by adding a type `AxiosResponse<Issue>`

Reply:
Neither of the API responses here are a simple map to the models - would you recommend I create classes for each in that case?

Reply:
ie this one returns 
`issues: {issues: Issue[], remaining: number, remaining_url: string}`